libs/net/dal/Models/ContentFilter.cs

```diff
@@ -173,6 +173,11 @@ public class ContentFilter : PageFilter
     /// </summary>
     public string[] Sort { get; set; } = Array.Empty<string>();
 
+    /// <summary>
+    /// get/set - The content sentitment.
+    /// </summary>
+    public long[] Sentiment { get; set; } = Array.Empty<long>();
```

Comment:
Is there any reason to use a `long`?  I think the value will never be more than -5 to +5.

Reply:
Copy and paste d'oh mistake, int now :)